apps/web/core/components/pages/projects/project-views/list-view/data-table.tsx

```
@@ -417,27 +416,15 @@ export const ProjectsTable = memo(
 			{
 				id: 'restore',
 				cell: function Cell({ row }) {
-					const {
-						openModal: openRestoreProjectModal,
-						closeModal: closeRestoreProjectModal,
-						isOpen: isRestoreProjectModalOpen
-					} = useModal();
+					const { openRestoreModal } = useProjectActionModal();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)